shells/shells.go

```diff
@@ -15,7 +16,24 @@ const (
 )
 
 func init() {
-	CurrentShell = models.Shell{Name: "fish", GetLastCmd: getLastLineFish}
+	CurrentShell = getShellFromProcess()
+}
+
+func getShellFromProcess() models.Shell {
+	var getLastLine = map[string]func() string{
```

Comment:
also we should probably initialize this somewhere else, maybe in init? it doesn't seem like something that should be function specific